validation/src/static/scripts/index.ts

```diff
@@ -1 +1,178 @@
+    getElem('ourAccName')!.innerText = '"' + accnames.accname + '"';
+
+    resultsTable.classList.remove('hidden');
+
+    if (comparisonResults[1]) {
+      const caseId = comparisonResults[1];
+
+      resultsContainer.innerHTML += `
+        <div class="bluetext comparisonResultText">A test-case was generated as a result of this comparison - <a class="linkBtn" href="/case/${caseId}">View Test-case ${caseId}</a></div>
+      `;
+    }
+  } else if (rawResponse.status === 400) {
+    const error = await rawResponse.json();
+    resultsContainer.innerHTML = `<div class="redtext comparisonResultText">Error: ${error.message}</div>`;
+  }
+};
+
+/**
+ * Sends call to run a URL comparison on provided input URL.
+ * Displays results of that comparison.
+ */
+const getURLComparison = async () => {
+  const urlInput = getElem('urlInput') as HTMLInputElement;
+  const urlResults = getElem('urlResults');
+  if (!urlInput || !urlResults) return;
+  urlResults.innerHTML = '';
+
+  // Adds an animated CSS Loading Spinner
+  const spinner = document.createElement('div');
+  spinner.classList.add('loader');
+  urlResults.append(spinner);
+
+  const url = urlInput.value;
+  const rawResponse = await fetch(
+    'http://localhost:3000/api/runURLComparison',
+    {
+      method: 'POST',
+      headers: {
+        Accept: 'application/json',
+        'Content-Type': 'application/json',
+      },
+      body: JSON.stringify({url: url}),
+    }
+  );
+
+  // Remove spinner once we get a response
+  spinner.remove();
+  if (rawResponse.status === 200) {
+    const summaryId = await rawResponse.json();
+    urlResults.innerHTML = `<div class="bluetext comparisonResultText">View the comparison results for ${url} - <a class="linkBtn" href="/summary/${summaryId}">Summary ${summaryId}</a></div>`;
+  } else if (rawResponse.status === 400) {
+    const error = await rawResponse.json();
+    urlResults.innerHTML = `<div class="redtext comparisonResultText">Error: ${error.message}</div>`;
+  }
+};
+
+/**
+ * Displays the contents of preview.json in the 'Comparison History' section.
+ * @param preview - the Preview to be displayed under 'Comparison History'
+ */
+function displayPreview(preview: Preview) {
+  const snippetContainer = getElem('snippetPreviews');
+  const summaryContainer = getElem('summaryPreviews');
+  if (!snippetContainer || !summaryContainer) return;
+
+  for (const snippet of preview.snippets) {
+    snippetContainer.innerHTML += `<a href="/case/${
+      snippet.caseId
+    }" class="invisibleLink"><div class="previewCard">Case ${
+      snippet.caseId
+    } <span>${snippet.role ? `(${snippet.role})` : ''}</span></div></a>`;
+  }
+
+  for (const summary of preview.pageSummaries) {
+    summaryContainer.innerHTML += `<a href="/summary/${
+      summary.urlSummaryId
+    }" class="invisibleLink"><div class="previewCard">${
+      summary.url
+    } <span>${Math.round(summary.percentDisagreement)}%</span></div></a>`;
```

Comment:
It's not obvious to me what this percent means without reading the code, I think it could be more clear. I'm not sure how you want to solve that in the UI (maybe a ❓ with a tooltip, maybe actual text on the screen, maybe you can think of something else...)